src/styles/login.css

```diff
@@ -0,0 +1,39 @@
+.login {
+  width: 30%;
+  margin: 0 auto;
+  padding: 5px 10px;
+  border: 1px solid var(--lightcrimson);
+}
+
+.login > form > label {
+  display: block;
+  font-size: 1.2rem;
+  margin: 8px 0;
+  color: #3863f3c3;
+  font-family: "Kanit", sans-serif;
```

Comment:
This should be in a variable, or on the body tag so you can use `font-family: inherit`.